### ISSUE TITLE
梯度爆炸，没有更改过任何参数

### ISSUE THREAD

**Issue by kevingarnetttwo** [xialeiliu/RankIQA#23]
准备好数据集之后,进行训练, loss = nan

I1108 13:14:31.719862 532190 solver.cpp:414]     Test net output #0: loss = 61.9594 (* 1 = 61.9594 loss)
I1108 13:14:32.594465 532190 solver.cpp:239] Iteration 0 (0 iter/s, 1.43691s/100 iters), loss = 60.7816
I1108 13:14:32.594534 532190 solver.cpp:258]     Train net output #0: loss = 60.7816 (* 1 = 60.7816 loss)
I1108 13:14:32.594565 532190 sgd_solver.cpp:112] Iteration 0, lr = 1e-05
I1108 13:15:43.635108 532190 solver.cpp:239] Iteration 100 (1.40769 iter/s, 71.0386s/100 iters), loss = nan
I1108 13:15:43.635473 532190 solver.cpp:258]     Train net output #0: loss = nan (* 1 = nan loss)
I1108 13:15:43.635491 532190 sgd_solver.cpp:112] Iteration 100, lr = 1e-05

**Comment by kevingarnetttwo** [comment]
自己解决了
